addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,223 +1,265 @@
-$gold: #ffcc00;
+$gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
 
 .website_forum {
+    .o_forum_ask_btn {
+        @include media-breakpoint-up(md) {
+            box-shadow: $box-shadow;
+            width: 200px;
+        }
+    }
+
+    .o_wforum_nav.nav.nav-pills .nav-link {
+        line-height: 1;
+        @include o-hover-text-color($body-color, $link-color);
+
+        .fa {
+            opacity: 0.5;
+        }
 
-    .question-block .question-name {
-        max-width: 80%;
+        &:hover, &.active {
+            .fa {
+                opacity: 1;
+            }
+        }
+
+        &.active {
+            background-color: rgba(theme-color('info'), 0.1);
+            color: darken(theme-color('info'), 15%)!important;
+        }
     }
 
-    .o_post_content {
+    // Single Post
+    .o_wforum_post, .note-editable.card-block {
+        word-wrap: break-word;
+
         pre {
             padding: 1.5rem;
             border-radius: 5px;
             color: $white;
             background-color: $gray-700;
         }
+
         blockquote {
             position: relative;
-            padding-left: 3rem;
-            border-left: 2px solid $gray-500;
-            font-style: italic;
-            font-weight: lighter;
+            padding-left: 1em;
+            border-left: .25em solid $gray-500;
+            color: $gray-500;
         }
-        word-wrap: break-word;
     }
 
-    .vote {
-        min-width: 38px;
-        margin-right: 12px;
-        .vote_count {
-            font-weight: bold;
-            line-height: 12px;
-            font-size: 160%;
-        }
-        .vote_down,
-        .vote_up {
-            font-size: 130%;
-        }
-    }
+    .o_wforum_readable {
+        max-width: 700px;
 
-    .post_to_validate {
-        overflow: hidden;
-        .validate_answer {
+        > p:empty:first-child:last-child {
```

Comment:
Somehow related to [this](https://github.com/odoo/odoo/pull/35749#discussion_r320653612), but not longer necessary because you can't ask questions without provide a description (I initially  allowed this in a prev implementation).
I'll remove this